From 218f1590de04f88b53147ad42cb45b04e5566c06 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 30 Jan 2018 14:33:28 +0800
Subject: [PATCH 09/10] ghostscript-native:fix disable-system-libtiff

Modify configure to add the check to make sure
ghostscrip could work while system-libtiff is
disabled.

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Updated to apply to ghostscript 9.21.
Signed-off-by: Joe Slater <joe.slater@windriver.com>

Rebase to 9.22
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.ac | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/configure.ac b/configure.ac
index 44a1ed8..ad0db94 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1259,6 +1259,7 @@ case "x$with_system_libtiff" in
 esac
 
 if test x"$SHARE_LIBTIFF" = x"0" ; then
+    if test -e $LIBTIFFDIR/configure; then
       echo "Running libtiff configure script..."
       olddir=`pwd`
       if ! test -d "$LIBTIFFCONFDIR" ; then
@@ -1272,6 +1273,10 @@ if test x"$SHARE_LIBTIFF" = x"0" ; then
       cd "$olddir"
       echo
       echo "Continuing with Ghostscript configuration..."
+    else
+      AC_MSG_NOTICE([Could not find local copy of libtiff.
+Disabling tiff output devices.])
+    fi
 fi
 
 AC_SUBST(SHARE_LIBTIFF)
-- 
1.8.3.1

